examples/operator/basic/README.md

```diff
@@ -1,50 +1,64 @@
 # Basic Example
 
-This example code is a super-basic one-file example of a very simple kubernetes custom resource operator using the SDK. It doesn't handle synchronization or missed events, and is extremely noisy due to not filtering updates.
+This example code is a super-basic one-file example of a very simple kubernetes custom resource operator with a
+Watcher or a Reconciler, using the SDK `simple` package.
 
 ## To Run
 
 ### Using the run script
+
 1. Make sure you have k3d and go1.18+ installed
-2. Run the following to create a local k3d cluster and start the operator:
+2. Run the following to create a local k3d cluster and start the operator (you can choose between watcher and reconciler):
     ```shell
-   $ ./run.sh
+   $ ./run.sh <watcher/reconciler>
     ```
 
 ### Manually
+
 Start a local kubernetes cluster or use a remote one to which you have permission to create CRD's and monitor them.
 Set your kube context to the appropriate cluster, then run the operator:
+
 ```shell
-$ go run basic.go --kubecfg="path_to_your_kube_config"
+$ go run <watcher/reconciler>/main.go --kubecfg="path_to_your_kube_config"
 ```
 
-
 You may see one each of an error message about failing to watch and list the custom resource,
 this is due to a slight delay between the custom resource being added, and the control plane knowing it exists.
 This will only happen on the first run, when the resource definition doesn't yet exist in your cluster.
 
 ## Usage
 
 The operator will monitor for changes to any BasicCustomResource in your cluster, and log them. To demonstrate, run:
+
 ```shell
 $ kubectl create -f example.yaml
 ```
-You should see a log line for the added resource. The "noisy" part comes next, as you'll see repeated log lines for near-constant updates by kubernetes itself. These are all metadata updates, and can be filtered in a practical setting. For examples of such filtering, see the [opinionated](../opinionated/README.md) or `/example/operator/boilerplate` examples.
+
+You should see a log line for the added resource. The "noisy" part comes next, as you'll see repeated log lines for
+near-constant updates by kubernetes itself. These are all metadata updates, and can be filtered in a practical setting.
+For examples of such filtering, see the [opinionated](../opinionated/README.md) or `/example/operator/boilerplate`
```

Comment:
Since `simple` wraps reconcilers and watchers with their opinionated variants, we may want to instead mention that the logic used in `simple` follows the `opinioned` variants, and introduce code that uses an operator _without_ opinionated logic if need be.

Perhaps this also indicates that `simple.Operator` should have some level of preferences on wrapping reconcilers/watchers, but I also don't want `simple` to become too complex.

Reply:
To better understand what you're suggesting: should move the example using `simple` package in the `opinionated` section, and use a different approach in the `simple` section which is not using anything having the `opinionated` variant?

Reply:
I think for the moment, it might be enough to point out that using the `simple` package automatically uses the `opinionated` logic, and re-name `basic` to `simple`. We could introduce a `basic` variant later that doesn't use the opinionated logic at all, but has to build an operator without `simple` (or using `simple` with some "don't-use-opinionated" option yet to be introduced).